test/unit/v1beta1/metricscollector/test_tfevent_metricscollector.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+import utils
+
+
+class TestTFEventMetricsCollector(unittest.TestCase):
+    def test_parse_file(self):
+
+        logs_dir = "./test/unit/v1beta1/metricscollector/testdata/tfevent-metricscollector/logs"
```

Comment:
> Is there any way to dynamically generate these log files with summaries during the test run ?

I think we can dynamically generate these log files in GitHub Actions since I manually generate these using [`tf-mnist-with-summaries/mnist.py`](https://github.com/kubeflow/katib/blob/c485b91a47c0d23baacb1c518e49e91e3fe5cc4c/examples/v1beta1/trial-images/tf-mnist-with-summaries/mnist.py). 
I would like to change this unit test so that it dynamically generates log files, thanks! @andreyvelich 

> Or in that case, our test infra should have Tensorflow installed ?

This unit test runs on GitHub Actions so I do not think we need to modify our test infra.

Reply:
I addressed at https://github.com/kubeflow/katib/pull/1731/commits/2fd2ef897696ebaf234169f1023cd8d1b6dea7f0.